exla/Makefile

```diff
@@ -61,16 +61,24 @@ else
 	NVCCFLAGS = $(CFLAGS)
 endif
 
+
+ifeq ($(MIX_BUILD_EMBEDDED), true)
+	XLA_EXTENSION_LIB_RPATH = "xla_extension/lib"
+else
+	XLA_EXTENSION_LIB_RPATH = "../xla_extension/lib"
```

Comment:
small fix needed due to the addition of the version to the EXLA lib compile path